G/GDAL/build_tarballs.jl

```diff
@@ -3,24 +3,24 @@
 using BinaryBuilder, Pkg
 
 name = "GDAL"
-upstream_version = v"3.10.3"
+upstream_version = v"3.11.0"
 # The version offset is used for two purposes:
 # - If we need to release multiple jll packages for the same GDAL
 #   library (usually for weird packaging reasons) then we increase the
 #   offset because we usually cannot release the same version twice.
 # - Minor versions of GDAL are usually binary incompatible because
 #   they increase the shared library soname. To encode this, we
 #   increase the major version number of the version offset.
-version_offset = v"2.0.0"
+version_offset = v"3.0.0"
 version = VersionNumber(upstream_version.major * 100 + version_offset.major,
                         upstream_version.minor * 100 + version_offset.minor,
                         upstream_version.patch * 100 + version_offset.patch)
 
 # Collection of sources required to build GDAL
 sources = [
     GitSource("https://github.com/OSGeo/gdal.git",
-        "ed29baf16d20bccbe331d809e0bafd7afde03360"),
-    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
+        "447eb5238bb6ef2837e68bf2ec742c64007b680b"),
+    FileSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
```

Comment:
Why this change?

Reply:
It is much faster because the tarball is only unpacked when necessary, and then is unpacked directly into the system directory where it needs to go.